src/DependencyInjection/CmfSonataAdminIntegrationExtension.php

```diff
@@ -16,19 +16,46 @@
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\HttpKernel\DependencyInjection\Extension;
 use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
+use Symfony\Cmf\Bundle\SonataAdminIntegrationBundle\DependencyInjection\Factory\AdminFactoryInterface;
+use Symfony\Cmf\Bundle\SeoBundle\CmfSeoBundle;
 
 /**
  * @author Maximilian Berghoff <Maximilian.Berghoff@mayflower.de>
+ * @author Wouter de Jong <wouter@wouterj.nl>
  */
 class CmfSonataAdminIntegrationExtension extends Extension
 {
+    /**
+     * @var AdminFactoryInterface[]
+     */
+    private $factories;
+
+    /**
+     * @param null|AdminFactoryInterface[] $factories A list of Admin factories
```

Comment:
yep, re-read it and i think i get it now. but then the phpdoc should be "Map of config key to factory for that section." or something similar, right? the key does have a meaning.

imho we should either not use keys or remote the getKey method from the factory interface - having both seems redundant to me.